.travis.yml

```diff
@@ -7,4 +7,5 @@ before_install:
   - sh -e /etc/init.d/xvfb start
 script:
   - npm install
+  - pub run dart_dev test --unit
```

Comment:
They might have just gotten swallowed by the integration tests since they take a bit longer. There are 21 integration tests and you added 2 tests, and that CI run had 23 passing tests, so I think they're running.